cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -170,6 +174,13 @@ insertShelleyBlock tracer lenv firstBlockOfEpoch blk lStateSnap details = do
         SyncFollowing -> 10
         SyncLagging -> 2000
 
+    rightToJust (Right a) = Just a
+    rightToJust _ = Nothing
```

Comment:
There's a version of this in `https://hackage.haskell.org/package/either` that's already floating around in package deps somewhere.